docs/lakehouse-iceberg-catalog.md

```diff
@@ -103,6 +103,21 @@ For other Iceberg OSS properties not managed by Gravitino like `client.security-
 Please set the `warehouse` parameter to `oss://{bucket_name}/${prefix_name}`. Additionally, download the [Aliyun OSS SDK](https://gosspublic.alicdn.com/sdks/java/aliyun_java_sdk_3.10.2.zip) and copy `aliyun-sdk-oss-3.10.2.jar`, `hamcrest-core-1.1.jar`, `jdom2-2.0.6.jar` in the `catalogs/lakehouse-iceberg/libs/` directory.
 :::
 
+
+#### Other storages
+
+For storages that are not inherently integrated into Gravitino Iceberg REST service, you can manage them effectively through custom catalog properties.
```

Comment:
What's the meaning of "inherently integrated"?

Reply:
updated